### PR TITLE
remove unnecessary `frames` property from `closeWindow`

### PR DESCRIPTION
**Pull request by bridiver**
cc @bbondy

### CODE REVIEW

File: js/components/main.js [hunk 1]
```diff
@@ -1002,11 +997,9 @@ class Main extends ImmutableComponent {
             sortedFrames.map((frame) =>
               <Frame
                 ref={(node) => { this.frames[frame.get('key')] = node }}
-                tabData={this.props.appState.get('tabs').find((tab) => tab.get('tabId') === frame.get('tabId'))}
```
Comment (bridiver) [inline js/components/main.js]: this was just a random prop that wasn't being used anywhwere

Comment (bbondy) [inline js/components/main.js]: strange, I bet it was at some point and not anymore.

**Comment by bbondy** [comment]
++ pls merge as long as tests pass

**Comment by NejcZdovc** [comment]
@bbondy test are failing, but they are failing for master as well, so it's not related to this change

File: js/components/main.js (hunk 1 above)
Comment (bsclifton) [inline js/components/main.js]: @bridiver I don't remember the original use-case, but I did use this when implementing the per-tab alerts (but then removed in favor of a distinct `messageBoxDetail` property)

**Pull request merged**